ts_src/index.ts

```diff
@@ -0,0 +1,197 @@
+import createHash = require('create-hash');
+import { pbkdf2 as pbkdf2Async, pbkdf2Sync as pbkdf2 } from 'pbkdf2';
+import randomBytes = require('randombytes');
+
+// use unorm until String.prototype.normalize gets better browser support
+import unorm = require('unorm');
+
+// import CHINESE_SIMPLIFIED_WORDLIST from '../wordlists/chinese_simplified.json';
+import CHINESE_SIMPLIFIED_WORDLIST = require('./wordlists/chinese_simplified.json');
```

Comment:
I wonder if there's a way to explicitly say this is a `string[]` type?...

It looks like my Atom IDE is showing the content of `ENGLISH_WORDLIST[0]` as `string`... but it would be nice if it was explicit (for anyone reading the source)

Reply:
I think it's impossible to do it directly because typescript handle json as module and it's syntax doesn't allow to declare type to imported module.

One workaround is 
```typescript
import chinese_simplified = require('./wordlists/chinese_simplified.json');
const CHINESE_SIMPLIFIED_WORDLIST: string[] = chinese_simplified;
```

If you specify any other type, compiler will emit an error.